### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by pujamudaliar**
Codacy badge link added in README.md file , which shows codacy code quality status and redirects to the codacy repository page.

### CODE REVIEW

**Review by mukul-seagate11 — APPROVED** [review]
looks good

**Comment by vvv** [comment]
> Update README.md

@pujamudaliar The patch does not conform to Hare's [patch requirements](https://github.com/Seagate/cortx-hare/blob/dev/rfc/9/README.md#22-patch-requirements).

> 2. A patch SHOULD be a minimal and accurate answer to exactly one identified and agreed problem.
>
> \[...\]
>
> 6. A patch commit message SHOULD consist of a single short (less than 50 characters) line stating the problem ("Problem: ...") being solved, followed by a blank line and then the proposed solution ("Solution: ...").

What problem is being solved?

cc @Saumya-Sunder

**Review by vvv — CHANGES_REQUESTED** [review]
> Update README.md

@pujamudaliar The patch does not conform to Hare's [patch requirements](https://github.com/Seagate/cortx-hare/blob/dev/rfc/9/README.md#22-patch-requirements).

> 2. A patch SHOULD be a minimal and accurate answer to exactly one identified and agreed problem.
>
> \[...\]
>
> 6. A patch commit message SHOULD consist of a single short (less than 50 characters) line stating the problem ("Problem: ...") being solved, followed by a blank line and then the proposed solution ("Solution: ...").

What problem is being solved?

cc @Saumya-Sunder

**Comment by mukul-seagate11** [comment]
@NeeravChoudhari, Can you please take this forward as we had already made a practice to add Codacy badges for all the repos as,

image

**Comment by pujamudaliar** [comment]
Problem: Codacy Badge needs to be included in README.md file , which provides code quality status for every commit in the repository.It also contains link to the repository in Codacy.
Solution: Codacy badge link is copied from the settings tab under Codacy Hare repository and pasted in README.md file.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+[![Codacy Badge](https://app.codacy.com/project/badge/Grade/0642a6f7d92542e39f2dd68139cd5586)](https://www.codacy.com/gh/Seagate/cortx-hare/dashboard?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=Seagate/cortx-hare&amp;utm_campaign=Badge_Grade)
```
Comment (pujamudaliar) [inline README.md:1]: Problem: Codacy Badge needs to be included in README.md file , which provides code quality status for every commit in the repository.It also contains link to the repository in Codacy.
Solution: Codacy badge link is copied from the settings tab under Codacy Hare repository and pasted in README.md file.

**Comment by vvv** [comment]
> @NeeravChoudhari, Can you please take this forward as we had already made a practice to add Codacy badges for all the repos as I dont understand why the concern is raised for Hare

@mukul-seagate11 I speak some English, you can contact me as well. 🙂

**Review by vvv — APPROVED** [review]

**Pull request merged**